Makefile

```diff
@@ -17,6 +17,8 @@ e2e/clean:
 e2e/%: docker-build
 	kind create cluster --name capsule --image kindest/node:$* --config ./e2e/kind.yaml --wait=120s \
 	&& kubectl taint nodes capsule-worker2 key1=value1:NoSchedule
+	helm upgrade --install --namespace metrics-system --create-namespace metrics-server bitnami/metrics-server \
```

Comment:
We need `helm repo add bitnami https://charts.bitnami.com/bitnami` here before installing `metrics-server`